DbTables/src/DbTableFactory.cc

```diff
@@ -57,6 +67,8 @@ mu2e::DbTable::ptr_t mu2e::DbTableFactory::newTable(std::string const& name) {
     return std::shared_ptr<mu2e::DbTable>(new mu2e::SimEfficiencies2());
   } else if (name == "CalRoIDMapDIRACToOffline") {
     return std::shared_ptr<mu2e::DbTable>(new mu2e::CalRoIDMapDIRACToOffline());
+  } else if (name=="CalRoIDMapOfflineToDIRAC") {
+    return std::shared_ptr<mu2e::DbTable>(new mu2e::CalRoIDMapOfflineToDIRAC());    
```

Comment:
adding this duplicate line looks like a mistake

Reply:
these are different tables, right?

Reply:
I think you accidentally copied two lines related to CalRoIDMapOfflineToDIRAC, so they are duplicates, the lines showing up green here